runtime/common/src/apis.rs

```diff
@@ -953,10 +953,10 @@ macro_rules! impl_runtime_apis_plus_common {
 							Ok(Location::parent())
 						}
 						fn worst_case_holding(_depositable_count: u32) -> XcmAssets {
-						// 100 fungibles
-							const HOLDING_FUNGIBLES: u32 = 100;
+							// 50 fungibles
+							const HOLDING_FUNGIBLES: u32 = 50;
```

Comment:
Bring down the number of assets in the holding register to 50 to avoid `ExceedsMaxMessageSize` when running report_holding benchmark.